test/python/circuit/test_scheduled_circuit.py

```diff
@@ -164,13 +164,21 @@ def test_transpile_circuit_with_custom_instruction(self):
                               instruction_durations=[('bell', [0, 1], 1000)])
         self.assertEqual(scheduled.duration, 1500)
 
-    def test_transpile_delay_circuit_without_scheduling_method_as_normal_circuit(self):
+    def test_transpile_delay_circuit_without_scheduling_method(self):
         qc = QuantumCircuit(2)
         qc.h(0)
         qc.delay(500, 1)
         qc.cx(0, 1)
-        transpiled = transpile(qc)
-        self.assertEqual(transpiled.duration, None)
+        transpiled = transpile(qc, backend=self.backend_with_dt)
+        self.assertEqual(transpiled.duration, 2260)
```

Comment:
Note the altered behavior here, requiring delays to trigger implicit scheduling is no longer backwards compatible @ajavadia @itoko

Reply:
How do you want this handled?

Reply:
Ah, the original test case is renamed to `test_transpile_delay_circuit_without_scheduling_method_or_durs` but git missed it...

Anyways, I personally think it's ok to change the behavior in these cases because they are more like fixing a bug in a spec itself and we cannot fix them without losing backwards compatibility. Adding note (as you've already done so, thanks!) would be sufficient.